src/openforms/formio/formatters/base.py

```diff
@@ -51,12 +51,25 @@ def normalise_value_to_list(self, component: ComponentT, value: Any):
     def join_formatted_values(
         self, component: Component, formatted_values: Iterable[str]
     ) -> str:
-        if self.as_html:
-            args_generator = ((formatted,) for formatted in formatted_values)
-            return format_html_join(self.multiple_separator, "{}", args_generator)
-        else:
+        if not self.as_html:
             return self.multiple_separator.join(formatted_values)
 
+        args_generator = ((formatted,) for formatted in formatted_values)
+        match component.get("type"):
+            case "textfield" | "select":
```

Comment:
Don't limit the `ul` html logic to textfield and select. All multiple components should be displayed as html list

Reply:
All components that can be multiple are now shown as html lists